app/models/document_capture_session.rb

```diff
@@ -0,0 +1,24 @@
+class DocumentCaptureSession < ApplicationRecord
+  include NonNullUuid
+
+  belongs_to :user
+
+  def load_result
+    DocumentCaptureSessionResult.load(result_id)
+  end
+
+  def store_result_from_response(doc_auth_response)
+    result = DocumentCaptureSessionResult.new(
+      id: generate_result_id,
+      success: doc_auth_response.success?,
+      pii: doc_auth_response.pii_from_doc,
+    )
+    result.unload
```

Comment:
what if we added `DocumentCaptureSessionResult.store` so that we can do all this in one method? Or do we have a reason for separating initialization + storage?

Reply:
No reason, I can change it up.

Reply:
Just pushed a commit to change this up

Reply:
sweeet ty